inference-engine/src/legacy_api/src/ngraph_ops/gru_sequence_ie.cpp

```diff
@@ -62,9 +64,12 @@ void op::GRUSequenceIE::validate_and_infer_types() {
     PartialShape output_shape_0{PartialShape::dynamic(3)};
     PartialShape output_shape_1{PartialShape::dynamic(2)};
     if (get_input_partial_shape(0).is_static()) {
-        size_t batch_size = get_input_partial_shape(0).get_shape()[0];
-        size_t seq_length = get_input_partial_shape(0).get_shape()[1];
-        output_shape_0 = Shape{batch_size, seq_length, m_hidden_size};
+        size_t batch_size = get_input_partial_shape(0).get_shape()[1 - m_seq_axis];
```

Comment:
Looks like it is not safe.
Does it mean that m_seq_axis must be equal to 0 or 1?

Reply:
Yes, only 1 or 0, other values lead to an issue in plug-ins

Reply:
It is not a specified attribute for internal use only, for now. It will probably be officially specified next release

Reply:
ticket 43346